src/components/Buildbaron/BuildBaronTable.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import styled from "@emotion/styled/macro";
+import Badge from "@leafygreen-ui/badge";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Table } from "antd";
+import { StyledLink } from "components/styles";
+import { BuildBaron, JiraTicket } from "gql/generated/types";
+import { getDateCopy } from "utils/string";
+
+const { blue, gray } = uiColors;
+
+const columns = [
+  {
+    title: "Related tickets from JIRA",
```

Comment:
Jira is actually not an acronym (I think it's short for Gojira) so this shouldn't be all caps

Reply:
thanks! I was also wondering about why it was all caps. I'll fix it.